extensions/wikia/Email/tests/EmailIntegrationTest.php

```diff
@@ -16,15 +19,24 @@ function setUp() {
 	 * Make sure all the images we're using exist
 	 */
 	public function testEmailImages() {
-		$this->markTestSkipped();
 
-		$icons = Email\ImageHelper::getIconInfo();
+		$this->setVignetteEnvToProd();
 
-		foreach ( $icons as $info ) {
-			$url = $info['url'];
-			$name = $info['name'];
+		foreach ( Email\ImageHelper::getIconInfo() as $iconInfo ) {
+			$url = $iconInfo['url'];
+			$name = $iconInfo['name'];
 			$response = HTTP::get( $url );
 			$this->assertTrue( $response !== false, "{$name} should return HTTP 200" );
```

Comment:
Adding a `$url` in the assert message should help debugging in case of a failed assertion.